src/Workspaces/Core/Portable/Workspace/DocumentActiveContextChangedEventArgs.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using Microsoft.CodeAnalysis.Text;
+using Roslyn.Utilities;
+
+namespace Microsoft.CodeAnalysis
+{
+    public class DocumentActiveContextChangedEventArgs : EventArgs
+    {
+        public Workspace Workspace { get; }
+        public SourceTextContainer SourceTextContainer { get; }
+        public DocumentId PreviousContextDocumentId { get; }
```

Comment:
This search does a good job of listing the `New / Old` pairings. 

http://source.roslyn.io/#q=Old